src/pages/careers.js

```diff
@@ -78,7 +78,14 @@ const OPEN_ROLES = [
       text: `Become a top contributor to the open-source software that Roadie is built on.`,
     },
   ],
-  [],
+  [
+    {
+      title() {
+        return <Link to="/careers/platform-engineer/">Platform Engineer</Link>;
+      },
+      text: `Come build Roadies platform`,
```

Comment:
I wonder can we talk about a benefit that the candidate will receive if they take the job here? This particular proposition doesn't jump out at me as extremely attractive.

Grammar wise, should it be "Roadie's platform"? I can never remember this rule.

Reply:
how about "take an active role in shaping the future of our platform"

Reply:
Sounds better to me.

Reply:
remedied to "Take an active role in shaping the future of our platform"